src/helpers/commonFunctions.tsx

```diff
@@ -0,0 +1,6 @@
+export const formatNumber = (value: string) => {
+  return new Intl.NumberFormat('en-GB', {
```

Comment:
extract `new Intl.NumberFormat()` to a constant, so you don't create object instance each time you call formatNumber() function